src/serac/physics/utilities/quadrature_data.hpp

```diff
@@ -200,12 +259,15 @@ class QuadratureData : public SyncableData {
  */
 template <>
 class QuadratureData<void> {
+  // Doesn't do anything
+public:
+  SERAC_HOST_DEVICE std::nullptr_t operator()(const int, const int) { return nullptr; }
 };
```

Comment:
the general case of `QuadratureData<T>::operator()` returns a `T`, but this one returns a different type. Should this special case be a specialization on `std::nullptr_t` instead of `void`?